ui/components/app/name/name-details/index.scss

```diff
@@ -7,4 +7,8 @@
   &__address .mm-text-field {
     padding-right: 5px;
   }
+
+  .mm-modal-body {
+    overflow-y: visible !important;
```

Comment:
What's this fix for specifically? I remember that the dropdown successfully overlapped the modal at least?

And could we use a more specific selector to avoid the need for `!important`?

Reply:
You are spot-on. I've added a comment to clarify. Also, !important; was not necessary so I've removed it..

Reply:
It would be great to get a bug ticket for this and tag the `team-design-system` team if possible @dbrans  🙏